event-processing/event-processing-template.yml

```diff
@@ -2024,3 +2047,124 @@ Resources:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/EventProcessorFunction-KBV"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSKBVAddressLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/EventProcessorFunction-KBVAddress"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSKBVFraudLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/EventProcessorFunction-KBVFraud"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSIPVLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/EventProcessorFunction-IPV"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSIPVPassLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/EventProcessorFunction-IPVPass"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSSPOTLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/EventProcessorFunction-SPOT"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSFireHoseLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/firehose"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSObfuscationLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/ObfuscationFunction"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  CSLSCleanserFunctionLogsSubscription:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsProduction
+    Properties:
+      LogGroupName:
+        Ref: "/aws/lambda/CleanserFunction"
+      FilterPattern: ""
+      DestinationArn: "{{resolve:ssm:CSLSLogsDestination}}"
+
+  FraudBucketCSLSPolicy:
+    Type: AWS::S3::BucketPolicy
+    Condition: IsProduction
+    Properties:
+      Bucket: !Ref FraudSplunkDeliveryFailureBucket
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: 'Allow'
+            Resource:
+              - !GetAtt FraudSplunkDeliveryFailureBucket.Arn
+              - !Sub '${FraudSplunkDeliveryFailureBucket.Arn}/*'
+            Principal:
+              AWS: "{{resolve:ssm:CSLSS3LambdaARN}}"
```

Comment:
I don't know what's in this SSM parameter, but to make sure: it should be the Lambda role ARN, not the ARN for the lambda itself.

Reply:
Yeah, that's correct. I have just shortened the parameter name